### PR TITLE
Disablemonitoring

### PR DESCRIPTION
**Pull request by ranum**
After our investigation we have decided to disable the monitoring solution by default and allow users to enable it if they wish.  This PR sets the proper enabled flags to false and provides documentation as to the necessary changes that need to be made to enable in the future.

### CODE REVIEW

**Review by ewill00 — APPROVED** [review]
LGTM

**Review by luisgarcc — CHANGES_REQUESTED** [review]
Hi David, the changes look good, but can you rebase on top of the latest CI changes to make sure the changes will work? 
Also, a nit. The PR name should be something more meaningful than the name of the branch. Thanks!

File: clinical-ingestion/helm-charts/alvearie-ingestion/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ To uninstall/delete the `ingestion` deployment:
 helm delete ingestion
 ```
 
-Deletion of charts doesn't cascade to deleting associated `PersistedVolume`s and `PersistedVolumeClains`s. 
+Deletion of charts doesn't cascade to deleting associated `PersistedVolume`s and `PersistedVolumeClains`s.
```
Comment (ibmatc) [inline clinical-ingestion/helm-charts/alvearie-ingestion/README.md:64]: typo - "PersistedVolumeClaims" m/n

**Review by ibmatc — APPROVED** [review]

**Review by luisgarcc — APPROVED** [review]
Thanks!

**Pull request merged**